Unix/http/httpcommon.c

```diff
@@ -13,6 +13,16 @@
 #include <base/memman.h>
 #include <pal/format.h>
 
+//  In OpenSSL 0.9.8 and OpenSSL 1.0.0 (https://github.com/Microsoft/ostc-openssl), SSL_OP_NO_TLSv1_1 and SSL_OP_NO_TLSv1_2 are not defined in the old ssh.h file.
+//  We need these both variables to determine if disable TLS1.1 and TLS1.2.
+//  This cuases that "NoTLSv1_1=true" and "NoTLSV_2=true"opptions don't work when you add these opitons in omiserver.conf.  This issue has been reported by our customer.
```

Comment:
Okay I've edited this and added some comments. How does this sound?

//  In the base versions of OpenSSL 0.9.8 and OpenSSL 1.0.0 (https://github.com/Microsoft/ostc-openssl), SSL_OP_NO_TLSv1_1 and SSL_OP_NO_TLSv1_2 are not defined in the ssl.h header file, but we need them in order to tell OpenSSL to disable these protocols on later versions of OpenSSL 0.9.8/1.0.0.
//  We need these both variables to disable TLS1.1 and TLS1.2 if the customer requests to do so.
// Ideally, we would just build our products against later versions of OpenSSL 0.9.8/1.0.0 which do contain those flags mentioned above, but this could end up being problematic. If we build against later versions of OpenSSL, then our products may not work for customers who have old versions of OpenSSL installed on their system due to symbol runtime resolution.
//  If these flags aren't defined during build time, then even if a customer is using a later version of OpenSSL that does support these flags, they will not be able to disable TLS v1_1 or v1_2. This was discovered by one of our customers.

Reply:
@johnkord Thanks, your comments are clear more!